pgjdbc/src/test/java/org/postgresql/test/TestUtil.java

```diff
@@ -796,17 +796,21 @@ public static void printResultSet(ResultSet rs) throws SQLException {
     return res;
   }
 
-  public static String join(List<String> list) {
+  public static String join(List<String> list, String sep) {
     StringBuilder sb = new StringBuilder();
     for (String s : list) {
       if (sb.length() > 0) {
-        sb.append('\n');
+        sb.append(sep);
       }
       sb.append(s);
     }
     return sb.toString();
   }
 
+  public static String join(List<String> list) {
+    return join(list, "\n");
+  }
+
```

Comment:
Not a big deal, however, I would suggest "high-level -> lower-level" ordering.

In other words, place methods in such an order that represents their usages in the "above" methods. It helps readers since typically, people read from top to bottom (even when they start reading from the middle of the file :) )